src/App.js

```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
+  const buttonsInfo = [
+    { text: 'AC', changeBg: false },
+    { text: '+/-', changeBg: false },
+    { text: '%', changeBg: false },
+    { text: '/', changeBg: true },
+    { text: '7', changeBg: false },
+    { text: '8', changeBg: false },
+    { text: '9', changeBg: false },
+    { text: 'x', changeBg: true },
+    { text: '4', changeBg: false },
+    { text: '5', changeBg: false },
+    { text: '6', changeBg: false },
+    { text: '-', changeBg: true },
+    { text: '1', changeBg: false },
+    { text: '2', changeBg: false },
+    { text: '3', changeBg: false },
+    { text: '+', changeBg: true },
+    { text: '0', changeBg: false },
+    { text: '.', changeBg: false },
+    { text: '=', changeBg: true },
+  ];
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better if you exported this array of buttons to a different file to keep your code more organized and modular